README.md

```diff
@@ -49,8 +49,16 @@ yarn start-renderer-dev
 #    front-end, using React components served by the renderer job.
 yarn start-main-dev
 
-# Build an installable release.
+# Once window number **3** reports "Compiled", you can exit all four threads by 
+#    hitting Ctrl-C.
```

Comment:
This could use some clarification. My normal workflow is to leave all four windows open while I'm developing and running the app, so that I get continuous feedback as I'm developing. I'm guessing I didn't explain that well enough? I can add comments to try to clarify. If you're just trying to build the app for release, then I don't know that you need any of steps 1-4. How does that match with your understanding as a developer new to the project?